spec/services/remote_source_file_spec.rb

```diff
@@ -5,14 +5,14 @@ class TestObject < ActiveFedora::Base
 end
 
 describe Hydra::Derivatives::RemoteSourceFile do
-  describe '.call' do
+  describe 'when called' do
```

Comment:
I'm not getting this change. Does rubocop yell about `describe '.METHOD' do` now? I wouldn't expect a `'when...'` string with a `describe` (tend to see that with `context`s).

Reply:
You're right, overzealous updates based on the context issue noted below. Changing back now.